app/scripts/modules/kubernetes/label/labelConfigurer.component.html

```diff
@@ -0,0 +1,25 @@
+<table class="table table-condensed packed tags" style="border-top: 2px solid white;">
+  <tr ng-repeat="view in $ctrl.annotationViews track by $index">
```

Comment:
I would rather see the annotation configurer become a more generic component (rather than have the label configurer extend it) - especially since `$ctrl.annotationViews` ends up here.